app/views/projects/samples/index.html.erb

```diff
@@ -20,7 +20,7 @@
         data: {
           turbo_stream: true,
           controller: "action-link",
-          action_link_required_value: 1
+          action_link_required_value: 1,
```

Comment:
The comma at the end of the line here and below (line 33 and 51) shouldn't be necessary

Reply:
The updated `erb-formatter` is automatically adding these trailing commas now. See https://github.com/nebulab/erb-formatter/pull/35

Reply:
Thanks!